conf.py

```diff
@@ -29,7 +29,7 @@
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.todo',]
+extensions = ['sphinx.ext.todo','rst2pdf.pdfbuilder']
```

Comment:
It is a sphinx extension and has to be installed additionally.
Will add,how to get it to work in README